utilities/mtv_migration.py

```
@@ -299,6 +307,19 @@ def execute_migration(
 
+    # Capture populate pod logs for copy-offload migrations
+    # This must happen immediately after migration completes, before MTV cleans up the pods
+    try:
+        migration_uid: str = _get_migration_uid(plan=plan)
+        capture_populate_pod_logs(
+            ocp_admin_client=ocp_admin_client,
+            namespace=target_namespace,
+            migration_uid=migration_uid,
+            fixture_store=fixture_store,
+        )
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)